Dockerfile.web-client

```diff
@@ -0,0 +1,29 @@
+# Use official Node.js image as the base image for web client
+FROM node:14 as web-client-builder
+
+# Set the working directory in the container
+WORKDIR /app
+
+# Copy package.json and package-lock.json to the working directory
+COPY package*.json ./
+
+# Install dependencies
+RUN npm install
+
+# Copy the rest of the application code to the working directory
+COPY . .
```

Comment:
The source of the web client and demo app can be found at https://github.com/juspay/hyperswitch-web.

So you'd have to either add this Dockerfile there, or add steps in this Dockerfile to clone the repository.